code/modules/mob/living/carbon/xenomorph/damage_procs.dm

```diff
@@ -118,6 +120,15 @@
 /mob/living/carbon/Xenomorph/adjustFireLoss(amount)
 	fireloss = CLAMP(fireloss + amount, 0, maxHealth - crit_health)
 
+/mob/living/carbon/Xenomorph/proc/process_rage_damage(damage)
+	if(!damage || !istype(src,/mob/living/carbon/Xenomorph/Ravager))
```

Comment:
`return damage` (if that makes sense) and redefine for `/mob/living/carbon/Xenomorph/Ravager/process_rage_damage(damage)`